.storybook/utils.js

```diff
@@ -108,9 +107,9 @@ const parseMarkup = string => {
     // If results remain in the array, get the attributes
     if (result.length > 1 && typeof result[2] === "string") {
       // Break the attributes apart using the spaces
-      let attr = result[2].trim().split(" ");
+      let attr = result[2].trim().match(/[\w|-]+="[^"]+"/g);
```

Comment:
Note: this split was losing attributes that had more than one value.  So `foo="bar bat baz"` would be split to`["foo=\"bar", "bat", "baz\""].  No good!